changelog/26172.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+agent: Agent will re-trigger auto auth if token used for rendering templates has been revoked, has exceeded the number of uses, or is a bogus value.
```

Comment:
We should change this from `feature` to `improvement` or use the feature name syntax, e.g.
`**Agent Auto-Auth Self-Healing**: ...`
To me I think this makes more sense as an improvement (customer requires no changes, it affects old features), but it's up to you!